tests/unit/scripts/test_completion_server.py

```diff
@@ -715,3 +716,16 @@ def _get_current_completion_script_version(_script, lang):
     out, err = capsys.readouterr()
     assert not out  # never write to stdout
     assert not err
+
+
+async def test_prereqs_and_outputs():
+    """Test the error cases for listing task prereqs/outputs.
+
+    The succeess cases are tested in an integration test (requires a running
+    scheduler).
+    """
+    # if no tokens are provided, no prereqs or outputs are returned
+    assert await _list_prereqs_and_outputs([]) == ([], [])
+
+    # if an invalid workflow is provided, we can't list anything
+    assert await _list_prereqs_and_outputs([Tokens(workflow='no-such-workflow')]) == ([], [])
```

Comment:
```suggestion
    assert await _list_prereqs_and_outputs(
        [Tokens(workflow='no-such-workflow')]
    ) == ([], [])
```

Reply:
Note, flake8 is not presently enforced for the tests, it probably should be, but that's another matter.

Reply:
I know. This is merely my opinion.